Makefile

```diff
@@ -25,9 +35,12 @@ check:
 install: _output/bin/$(LOCAL_OS)/bootkube
 	cp $< $(GOPATH_BIN)
 
+_output/bin/%: GOOS=$(word 1, $(subst /, ,$*))
+_output/bin/%: GOARCH=$(word 2, $(subst /, ,$*))
+_output/bin/%: GOARCH:=amd64
```

Comment:
_output/OS/bootkube, _output/OS/checkpointer, and release tarball continue to default to amd64 to avoid breaking any of the other scripts in the repo or users of the tarball.

Reply:
Can you add this as a comment just to keep that clear the intention here.

Reply:
added